### PR TITLE
RNMobile: Image Block vol 02

### PR DESCRIPTION
**Pull request by etoledom**
## Description
This PR adds the ability to recognize an empty image block and one with content, showing the appropriate UI in each case.

This was implemented following the web version as an example, trying to recreate the same structure on mobile.

The UI is temporary and most of the code will probably change in the near future.

For now, the image is being downloaded directly by the `<Image>` component, but in the future, it might be better to let the main app to handle both downloads and uploads.

## Screenshots 

image_block_with_image

For testing, please refer to the [Gutenberg Mobile PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/143)

cc @hypest

### CODE REVIEW

File: packages/block-library/src/image/edit.native.js [hunk 1]
```diff
@@ -1,11 +1,38 @@
...
+/**
+ * Internal dependencies
+ */
+import { MediaPlaceholder } from '@wordpress/editor';
+
+export default function ImageEdit( { attributes } ) {
+	const { url } = attributes;
+
+	const onUploadPress = () => {
+	};
```
Comment (mzorz) [inline packages/block-library/src/image/edit.native.js]: Minor nit: I'd add some `console.log()` here to indicate this has been pressed, and also we could use a `TODO` comment in code in the function body to mark this as something we need to implement

**Review by mzorz — APPROVED** [review]
Works as advertised, nice work @etoledom! I left a minor comment but otherwise approving this work

File: packages/block-library/src/image/edit.native.js (hunk 1 above)
Comment (etoledom) [reply packages/block-library/src/image/edit.native.js]: `eslint` seems to not like console.logs:
`error  Unexpected console statement  no-console`
But I'll add the `TODO` comment for future references.

Thank you!

Comment (mzorz) [reply packages/block-library/src/image/edit.native.js]: Oh! 😅 makes sense 👍 thank you!

**Pull request merged**